lua/gatekeeper/policylib.lua

```diff
@@ -281,3 +281,40 @@ function decision_web(policy, tx_rate_kib_sec, cap_expire_sec,
 		policy, tx_rate_kib_sec, tx_rate_kib_sec * 0.05, -- 5%
 		cap_expire_sec, next_renewal_ms, renewal_step_ms, false)
 end
+
+local function lpm_add_tbl8s(ip_addr, prefix_len, prefixes)
+	local masked_ip
```

Comment:
While `lpm_para_estimate()` is specific to the format of the file read, `lpm_add_tbl8s()` is generic and certainly reusable in other policies. Could you move `lpm_add_tbl8s()` to `lpmlib`?

Do the same to `lpm6_add_tbl8s()` in the IPv6 patch.

Reply:
If we move ``lpm_add_tbl8s()`` to ``lpmlib``, do we still need to keep ``ip_mask_addr()`` in ``lpmlib``?

Reply:
Thanks for the clarification.